lib/cfonb/operation_details.rb

```diff
@@ -15,7 +15,9 @@ def self.register(code, klass)
     end
 
     def self.for(line)
-      @details[line.detail_code]
+      return unless line.respond_to?(:detail_code)
```

Comment:
We do that below, this was to prevent cases where the file might be corrupted and we didn't have a detail_code at all (since we need the detail code for both know known and unknown classes). Maybe I should raise instead?